BoldGrid/crio
# Onboarding videos

# Add Tutorial Videos to Customizer #

## Test Files ##

[crio-2.22.0-rc.1.zip](https://github.com/BoldGrid/crio/files/13942214/crio-2.22.0-rc.1.zip)

## NOTES TO TESTER ##
### Please leave comments regarding issues found in testing directly on the issue linked above, not the Pull Request. This helps ease the process of reviewing the testing of multiple PRs in a given project from the project view. ###
### Please remember to move this item from 'Needs Review' to either 'In Progress' or 'Awaiting Release' depending on the results of your testing ###

## Testing Instructions ##

1. Install BoldGrid Inspirations, and proceed through the Inspirations deployment process.
2. Upload the included zip file to replace the installed version of Crio.
3. Open the Customizer.
4. Once the Customizer loads, you should see a modal appear showing the list of videos.
5. Ensure all buttons work and videos play.
6. On closing the modal, ensure that an admin pointer appears, directing you to where you can re-open the videos.
7. Reload the customizer.
8. Ensure that the modal does not automatically display again.
